src/global_mapper/global_mapper_params.h

```diff
@@ -8,16 +8,32 @@ class Params {
  public:
   Params();
 
-  std::vector<double> voxel_xyz_min_;
-  std::vector<double> voxel_xyz_max_;
-  std::vector<double> voxel_resolution_;
+  // pixel map
+  double pixel_xy_min_[2];
+  double pixel_xy_max_[2];
```

Comment:
Great point.  I hadn't realized that was actually in the GSG.  I switched params over to a struct, removed the underscores, and removed the source file so I don't have to keep editing everything in two places.  Trying to lower the barrier to adding a param so I do it more often.